dev/io.openliberty.io.smallrye.reactive.messaging-provider4/src/io/smallrye/reactive/messaging/providers/extension/ReactiveMessagingExtension.java

```diff
@@ -0,0 +1,259 @@
+package io.smallrye.reactive.messaging.providers.extension;
```

Comment:
Given you have done the work required to remove this overlay, we should not be readding it